fix torch rec test failure

ydwu4 opened the pull request:
Summary: Fixes T192448049. The module call form an unusal call stack for the nodes: https://www.internalfb.com/phabricator/paste/view/P1507230978. This is currently not supported by unflattener and need some extra design to make it work.

Test Plan: buck2 run 'fbcode//mode/opt' torchrec/distributed/tests:test_pt2 -- --filter-text "test_sharded_quant_fpebc_non_strict_export"

Reviewed By: zhxchen17

Differential Revision: D60528900

Skylion007 commented on torch/export/unflatten.py:
```diff
@@ -594,9 +594,13 @@ def _compute_accessor(parent_fqn: str, child_fqn: str) -> str:
+    # TODO: support skip connection by inlining the child module.
+    if child_split[: len(parent_split)] != parent_split:
+        raise RuntimeError(
+            "Child module '%s' is not a descendant of parent mldule '%s'."
```
This has a lot of typos and the string placeholders aren't filled in for some reason? Why did we remove the fstrings?

ydwu4 replied:
Oops, it's by accident. Will fix that

ydwu4 committed:
[export] Added an un-supported pattern of unflatten (#132437)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/132437

Added a todo for an un-supported pattern of unflatten

Test Plan: new test.

Reviewed By: zhxchen17

Differential Revision: D60528900
